home/lncm/box-install.sh

```diff
@@ -16,191 +16,191 @@ SD=mmcblk0
 START=268M
 
 check_installed() {
-  if [ -f /media/${SD}p1/installed ]; then
-    #  installed
-    echo 0
-    return 0
-  else
-    # not installed
-    echo 1
-    return 1
-  fi
+	if [ -f /media/${SD}p1/installed ]; then
+		#  installed
+		echo 0
+		return 0
+	else
+		# not installed
+		echo 1
+		return 1
+	fi
 }
 
 install_apk() {
-  echo "Install $1"
-  while [ -z "$( apk -e info $1 )" ]; do
-    /sbin/apk add $1
-  done
+	echo "Install $1"
+	while [ -z "$(apk -e info $1)" ]; do
+		/sbin/apk add $1
+	done
 }
 
 mnt_rw() {
-  echo "Re-mount FAT partition read/write"
-  /bin/mount -o remount,rw /dev/"${SD}"p1 /media/"${SD}"p1
+	echo "Re-mount FAT partition read/write"
+	/bin/mount -o remount,rw /dev/"${SD}"p1 /media/"${SD}"p1
 }
 
 mnt_ro() {
-  echo "Re-mount FAT partition read-only"
-  /bin/mount -o remount,ro /dev/"${SD}p1" /media/"${SD}"p1
+	echo "Re-mount FAT partition read-only"
+	/bin/mount -o remount,ro /dev/"${SD}p1" /media/"${SD}"p1
 }
 
 install_tools() {
-  echo "Add partitioning and filesystem tools"
-  /sbin/setup-apkcache /media/mmcblk0p1/cache
-  install_apk parted
-  install_apk e2fsprogs
+	echo "Add partitioning and filesystem tools"
+	/sbin/setup-apkcache /media/mmcblk0p1/cache
+	install_apk parted
+	install_apk e2fsprogs
 }
 
 partition_sd() {
-  echo "Create second primary partition"
-  /usr/sbin/parted -s /dev/$SD mkpart p ext4 $START 100%
+	echo "Create second primary partition"
+	/usr/sbin/parted -s /dev/$SD mkpart p ext4 $START 100%
 }
 
 create_ext4() {
-  echo "Create and mount ext4 filesystem"
-  /sbin/mkfs.ext4 -F /dev/${SD}p2 && \
-  /bin/echo "/dev/${SD}p2 /media/sd ext4 noatime 0 0" >> /etc/fstab && \
-  /bin/mkdir /media/sd && \
-  /bin/mount /media/sd
+	echo "Create and mount ext4 filesystem"
+	/sbin/mkfs.ext4 -F /dev/${SD}p2 &&
```

Comment:
Yyyyyy… Are you sure that the removal of the slashes here doesn't break the code?